### DISCUSSION TITLE
Sidekick help

### DISCUSSION THREAD

**Discussion by djxxx77** [frntc/Sidekick64#84] [General]
Just got a sidekick recently and everything works apart from tft screen doesn’t come up when I switch on… the only way I can get it to come on is to press f1 for georom then go into basic. Hold rest button down for about 5 seconds and the screen appear… is there any other config files I have to set as I’m only using the one in c64 directory

**Comment by frntc** [comment]
I think we talked on Lemon64.

For those who might be interested: the problem was bad soldering at the socket for the TFT.
